controls/os_spec.rb

```diff
@@ -142,9 +179,24 @@
     '/usr/lib/pt_chown',                                          # pseudo-tty, needed?
     '/usr/lib/eject/dmcrypt-get-device',
     '/usr/lib/mc/cons.saver' # midnight commander screensaver
+#    # from Ubuntu xenial
+#    '/sbin/unix_chkpwd',
+#    '/sbin/pam_extrausers_chkpwd',
+#    '/usr/lib/x86_64-linux-gnu/utempter/utempter',
+#    '/usr/sbin/postdrop',
+#    '/usr/sbin/postqueue',
+#    '/usr/bin/ssh-agent',
+#    '/usr/bin/mlocate',
+#    '/usr/bin/crontab',
+#    '/usr/bin/dotlockfile',
+#    '/usr/bin/screen',
+#    '/usr/bin/expiry',
+#    '/usr/bin/wall',
+#    '/usr/bin/chage',
+#    '/usr/bin/bsd-write',
   ]
 
-  output = command('find / -perm -4000 -o -perm -2000 -type f ! -path \'/proc/*\' -print 2>/dev/null | grep -v \'^find:\'')
+  output = command('find / -perm -4000 -o -perm -2000 -type f ! -path \'/proc/*\' ! -path \'/var/lib/lxd/containers/*\' -print 2>/dev/null | grep -v \'^find:\'')
```

Comment:
this directory contains root of the multiple lxc containers active on the system = will repeat totally or not usual system

example
```
# find / -perm -4000 -o -perm -2000 -type f ! -path '/proc/*' -print 2>/dev/null | grep -v '^find:'
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/ssh-agent
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/expiry
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/mlocate
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/screen
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/bsd-write
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/chage
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/wall
/var/lib/lxd/containers/default-ubuntu-1604-1481820784/rootfs/usr/bin/crontab
...
```

ideally an option depending on what's your perspective on container: audit them from host with the host, from host with separate config, or inside the container.